README.md

```diff
@@ -1,6 +1,6 @@
 # ![Foxbox Client](https://raw.githubusercontent.com/fxbox/app/master/app/img/icons/32.png "Foxbox Client") Foxbox Client
 
-> A client for Foxbox
+> A web client for Foxbox (with support for emulating the server component in the meantime).
```

Comment:
Right, in that case I think it might be confusing to state that explicitly, since it's not really that our app support it, more that `gulp` does, and all client-side javascript build systems offer that functionality (not only `gulp` but also `grunt` and `bower` IIRC).

Also, as we're about to add a Cordova build target, we could say something "A web client for Foxbox" and leave out the remark about `gulp`'s static webserver support.